src/main/resources/static/css/style.css

```diff
@@ -1,52 +1,126 @@
-@charset "ISO-8859-1";
-
-h1 { 
-	color: #7c795d; 
-	font-family: 'Trocchi', serif; 
-	font-size: 45px; 
-	font-weight: normal; 
-	line-height: 48px; 
-	margin: 0; 
-	text-align: center;}
-
-h2 { 
-	color: #111; 
-	font-family: 'Open Sans', sans-serif; 
-	font-size: 30px; 
-	font-weight: 300; 
-	line-height: 32px; 
-	margin: 0 0 72px; 
-	text-align: center; }
-	
-h3 {
-	color: #111; 
-	font-family: 'Open Sans', sans-serif; 
-	font-size: 30px; 
-	font-weight: 300; 
-	line-height: 32px; 
-	margin: 0 0 72px; 
-	text-align: center;
-	
-}
-
-button {
-  display: inline-block;
-  padding: 15px 25px;
-  font-size: 24px;
-  cursor: pointer;
-  text-align: center;
-  text-decoration: none;
-  outline: none;
-  color: #ffffff;
-  background-color: #393937;
-  border: none;
-  border-radius: 15px;
-  box-shadow: 0 9px #999;
-}
-
-button:hover {background-color: #3e8e41}
-button:active {
-  background-color: #3e8e41;
-  box-shadow: 0 5px #666;
-  transform: translateY(4px);
-}
\ No newline at end of file
+body {

```

Comment:
updated CSS with the new design. Feel free to reverse it back to the original if you don't like it. You can find a lot of CSS + HTML idea in `https://codepen.io`